internal/deviations/deviations.go

```diff
@@ -597,14 +597,14 @@ func ISISCounterPartChangesUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIsisCounterPartChangesUnsupported()
 }
 
-// TransceiverThresholdsUnsupported returns true if the device does not support threshold container under /components/component/transceiver.
-// Default value is false.
-func TransceiverThresholdsUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
What do you want to do about the existing test using this deviation?
https://github.com/openconfig/featureprofiles/blob/24d10056e7453a2750439d7a329a4c02ba838616/feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go#L97